sputnikdao2/src/policy.rs

```diff
@@ -269,8 +275,21 @@ impl Policy {
         env::log_str(&format!("ERR_ROLE_NOT_FOUND:{}", role));
     }
 
-    /// Returns set of roles that this user is memeber of permissions for given user across all the roles it's member of.
-    fn get_user_roles(&self, user: UserInfo) -> HashMap<String, &HashSet<String>> {
+    /// Removes `member_id` from all roles.  
+    /// Returns `true` if the member was removed from at least one role.
+    pub fn remove_member_from_all_roles(&mut self, member_id: &AccountId) -> bool {
+        let mut removed = false;
+        for role in self.roles.iter_mut() {
+            if let RoleKind::Group(ref mut members) = role.kind {
+                removed |= members.remove(member_id);
```

Comment:
bitwise or?

Reply:
That's supposed to be a shortcut to `removed = removed | /*..*/;` 
But bools end up using boolean logic instead of bitwise operations